features/bevragen/zoek-met-nummeraanduiding-identificatie/fout-cases.feature

```diff
@@ -0,0 +1,96 @@
+#language: nl
+
+Functionaliteit: Zoek met nummeraanduiding identificatie - fout cases
+
+Rule: nummeraanduidingIdentificatie is een verplichte parameter
+
+  @fout-case
+  Scenario: De nummeraanduidingIdentificatie parameter is niet opgegeven
+    Als personen wordt gezocht met de volgende parameters
+    | naam   | waarde                               |
+    | type   | ZoekMetNummeraanduidingIdentificatie |
+    | fields | burgerservicenummer                  |
+    Dan heeft de response een object met de volgende gegevens
+    | naam     | waarde                                                        |
+    | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1   |
+    | title    | Minimale combinatie van parameters moet worden opgegeven.     |
+    | status   | 400                                                           |
+    | detail   | De foutieve parameter(s) zijn: nummeraanduidingIdentificatie. |
+    | code     | paramsCombination                                             |
+    | instance | /haalcentraal/api/brp/personen                                |
+    En heeft het object de volgende 'invalidParams' gegevens
+    | code     | name                          | reason                  |
+    | required | nummeraanduidingIdentificatie | Parameter is verplicht. |
+
+  @fout-case
+  Scenario: Een lege string is opgegeven als nummeraanduiding identificatie waarde
+    Als personen wordt gezocht met de volgende parameters
+    | naam                          | waarde                               |
+    | type                          | ZoekMetNummeraanduidingIdentificatie |
+    | nummeraanduidingIdentificatie |                                      |
+    | fields                        | burgerservicenummer                  |
+    Dan heeft de response een object met de volgende gegevens
+    | naam     | waarde                                                        |
+    | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1   |
+    | title    | Minimale combinatie van parameters moet worden opgegeven.     |
+    | status   | 400                                                           |
+    | detail   | De foutieve parameter(s) zijn: nummeraanduidingIdentificatie. |
+    | code     | paramsCombination                                             |
+    | instance | /haalcentraal/api/brp/personen                                |
+    En heeft het object de volgende 'invalidParams' gegevens
```

Comment:
Is de waarde 'Parameter is verplicht.' voor reason logisch in deze situatie? In principe wordt de parameter nummeraanduidingIdentificatie immers meegegeven maar met een verkeerde waarde.
De reason 'Waarde voldoet niet aan patroon ^[0-9]{16}$.' lijkt mij logischer.